src/exposurelog/routers/delete_messages.py

```diff
@@ -0,0 +1,53 @@
+from __future__ import annotations
+
+__all__ = ["delete_messages"]
+
+import typing
+
+import astropy.time
+import fastapi
+import sqlalchemy as sa
+
+from ..message import Message
+from ..shared_state import SharedState, get_shared_state
+
+router = fastapi.APIRouter()
+
+
+@router.post(
```

Comment:
I'm not a REST expert but when we are deleting something shouldn't we be using DELETE rather than POST?

Reply:
It turns out that DELETE doesn't accept parameters, so I think it's a non-starter here -- though perhaps I could provide the params as Query params instead of Form params. In any case POST works and it turns out this is not an idiomatic REST API (though it is still stateless) so I am withdrawing any mention of REST.

I am considering using a more idiomatic REST interface, but want to think about it a bit. I am not at all clear on what the advantages would be.